com.github.debauchee.barrier.appdata.xml

```diff
@@ -0,0 +1,104 @@
+
+  Project goals
+
+  Hassle-free reliability. We are users, too. Barrier was created so that we could solve the issues we had with synergy and then share these fixes with other users.
+
+  Compatibility. We use more than one operating system and you probably do, too. Windows, OSX, Linux, FreeBSD... Barrier should "just work". We will also have our eye on Wayland when the time comes.
+
+  Communication. Everything we do is in the open. Our issue tracker will let you see if others are having the same problem you're having and will allow you to add additional information. You will also be able to see when progress is made and how the issue gets resolved.
+
+  Contact &amp; support
+
+  Please be aware that the only way to draw our attention to a bug is to create a new PR in the issue tracker. Write a clear, concise, detailed report and you will get a clear, concise, detailed response. Priority is always given to issues that affect a wider range of users.
+
+  For short and simple questions or to just say hello find us on the Freenode IRC network in the #barrier channel.
+
+  Contributions
+
+  At this time we are looking for developers to help fix the issues found in the issue tracker. Submit pull requests once you've polished up your patch and we'll review and possibly merge it.
+  </description>
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apa.html -->
+  <!-- https://specifications.freedesktop.org/menu-spec/latest/apas02.html -->
+  <categories>
+    <category>Network</category>
+  </categories>
+  <url type="homepage">https://github.com/debauchee/barrier/</url>
+  <url type="bugtracker">https://github.com/debauchee/barrier/issues/</url>
+  <launchable type="desktop-id">com.github.debauchee.barrier.desktop</launchable>
+ 
+  <releases>
+    <release date="2019-01-11" version="2.1.2" urgency="medium" />
+    <release date="2018-07-02" version="2.1.1" urgency="medium" />
+  </releases>
+ 
+  <project_group>Debauchee</project_group>
+  <project_license>GPL-2.0</project_license>
+  <developer_name>Siix</developer_name>
+
+  <translation type="gettext">avahi</translation>
+  <launchable type="desktop-id">barrier.desktop</launchable>
+
+  <screenshots>
+    <screenshot type="default">
+      <caption>The main application window</caption>
+      <image type="source">https://i.imgur.com/hTlK5Sp.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>The Server Configuration window</caption>
+      <image type="source">https://i.imgur.com/JmzUSbh.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>The Settings menu</caption>
+      <image type="source">https://i.imgur.com/YueKkh4.png</image>
+    </screenshot>
+  </screenshots>
+  
+  <update_contact>adrianlucrececeleste@airmail.cc</update_contact>
+ 
+  <content_rating type="oars-1.1">
```

Comment:
you don't actually need the none parts any more - you can just use `<content_rating type="oars-1.1" />`